packages/react-core/src/components/Toolbar/examples/Toolbar.md

```diff
@@ -390,6 +393,272 @@ class ToolbarGroupTypes extends React.Component {
+      });
+    };
+    
+    this.onSearchNext = (event) => {
+      this.setState(prevState => {
+        const newCurrentResult = prevState.currentSearchResult + 1;
+        return {
+          currentSearchResult: newCurrentResult <= prevState.searchResultsCount ? newCurrentResult : prevState.searchResultsCount
+        } 
+      });
+    };
+    
+    this.onSearchPrevious = (event) => {
+      this.setState(prevState => {
+        const newCurrentResult = prevState.currentSearchResult - 1;
+        return {
+          currentSearchResult: newCurrentResult > 0 ? newCurrentResult : 1 
+        }
+      });
+    };
+  }
+
+  render() {
+    const {
+      firstIsExpanded,
+      firstSelected,
+      isPaused,
+      firstSwitchChecked,
+      secondSwitchChecked,
+      searchValue,
+      searchResultsCount,
+      currentSearchResult,
+      secondIsExpanded,
+      thirdIsExpanded
+    } = this.state;
+
+    const firstDropdownItems = [
+      <DropdownItem key="action" component="button" onClick={this.onSecondClick}>
+        External Logs
+      </DropdownItem>,
+      <DropdownItem key="action" component="button" onClick={this.onSecondClick}>
+        External Logs
+      </DropdownItem>,
+      <DropdownItem key="action" component="button" onClick={this.onSecondClick}>
+        External Logs
+      </DropdownItem>
+    ];
+
+    const secondDropdownItems = [
+      <DropdownItem key="action" component="button" onClick={this.onThirdClick}>
+        Export
+      </DropdownItem>,
+      <DropdownItem key="action" component="button" onClick={this.onThirdClick}>
+        Clear
+      </DropdownItem>
+    ];
+
+    const items = (
+      <React.Fragment>
+        <ToolbarItem>
+          <Select
+            aria-label="Select Input"
+            onToggle={this.onFirstToggle}
+            onSelect={this.onFirstSelect}
+            selections={firstSelected}
+            isOpen={firstIsExpanded}
+          >
+            {this.firstOptions.map((option, index) => (
+              <SelectOption isDisabled={option.disabled} key={index} value={option.value}>
+                <Badge key={index}>C</Badge>
+                {` ${option.value}`}
+              </SelectOption>   
+            ))}
+          </Select>
+        </ToolbarItem>
+        <ToolbarItem>
+          <Button variant="link" onClick={this.pauseOrStart}>
+          {this.state.isPaused? <PlayIcon /> : <PauseIcon />}
+          {` Log Stream`}
+          </Button>
+        </ToolbarItem>
+        <ToolbarItem>
+          <Switch
+            id="switch-1"
+            label="Timestamp"
+            isChecked={firstSwitchChecked}
+            onChange={this.handleFirstSwitchChange}
+          />
+        </ToolbarItem>
+        <ToolbarItem>
+          <Switch
+            id="switch-2"
+            label="Wrap lines"
+            isChecked={secondSwitchChecked}
+            onChange={this.handleSecondSwitchChange}
+          />
+        </ToolbarItem>
+        <ToolbarItem>
```

Comment:
According to the design: http://openshift.github.io/openshift-origin-design/designs/administrator/future-openshift/log%20toolbars/

The wrapped lines marks the end of the first group of components on the toolbar which are left justified, and the last group are right justified. @nicolethoen might be able to help here, she showed us how it could be done during a meeting about this component.

This is how the component currently looks at a larger size format:

<img width="1703" alt="Screen Shot 2020-11-30 at 4 39 43 PM" src="https://user-images.githubusercontent.com/35978579/100668949-ad0ebf80-332a-11eb-9c11-8f4a68553710.png">

Reply:
Yeah I believe there is an `alignment` prop that can go on the `ToolbarContent`, the `ToolbarGroup`, or the `ToolbarItem` depending on the structure. And it can set the alignment at specific or all viewport widths.

Reply:
I think you'll want to group the left aligned items and the right aligned items separately in different ToolbarGroups